dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactoryImpl.java

```diff
@@ -317,17 +318,38 @@ private boolean isValidContainerPath(final Folder folder) {
 
 	private List<Container> findHostContainers(final Host host, final User user) throws DotDataException, DotSecurityException {
 
-		// todo: replace this for ES call, not matter the host by sort by host
-		final List<Container> containers = new ArrayList<>();
-		final Optional<Folder> folder = this.findContainerFolderByHost(host, user);
+		final List<Folder> subFolders = this.findContainersAssetsByHost(host, user);
+		return this.getFolderContainers(host, user, subFolders);
+	}
+
+
+	private List<Folder> findContainersAssetsByHost(final Host host, final User user) {
+
+		List<Contentlet>           containers = null;
+		final List<Folder>         folders    = new ArrayList<>();
+
+		try{
 
-		if (folder.isPresent() && UtilMethods.isSet(folder.get().getIdentifier())) {
+			final String query = builder("+structureType:", Structure.STRUCTURE_TYPE_FILEASSET,

```

Comment:
Do we care at this point of the status of the `container.vtl` file?, meaning `+live:true`?

Reply:
great point - yes we need to pass +working:true and then take state and permission into consideration when you try to load the file based container .